k8s/client-deployment.yaml

```
@@ -0,0 +1,166 @@
+        runAsNonRoot: true
+        runAsUser: 101  # nginx user
+        fsGroup: 101
+      containers:
+      - name: blog-client
+        image: blog-app/client:latest
+        imagePullPolicy: Always
+        ports:
+        - name: http
+          containerPort: 80
+          protocol: TCP
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)